aws-lambda-layer/Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM lambci/lambda:build-ruby2.5
+FROM amazonlinux:2
 
 ARG FREETDS_VERSION=1.1.6
 
 ENV INSTALLDIR='/tmp/freetds'
 ENV FREETDS_VERSION=$FREETDS_VERSION
 
 RUN yum update -y
-RUN yum install wget build-essential libc6-dev -y
+RUN yum install wget build-essential libc6-dev tar gzip install -y
+RUN yum group install "Development Tools" -y
```

Comment:
What's in the "development tools" group that's missing from the explicit list (and vice versa?)... I'm wondering if we need them both, or if it'd be better to consolidate on one approach or the other

Reply:
I don't know which package is missing but I couldn't compile freetds without the `Development Tools` , I think I can test it and extract only the needed ones and consolidate it all in a single `RUN` install command if it's possible.